src/components/avatar/index.tsx

```diff
@@ -318,14 +349,48 @@ class Avatar extends PureComponent<AvatarProps> {
     }
   }
 
+  getText = memoize((label, name) => {
+    let text = label;
+    if (_.isNil(label) && !_.isNil(name)) {
+      text = AvatarHelper.getInitials(name);
+    }
+
+    return text;
+  });
+
+  get text() {
+    const {label, name} = this.props;
+    return this.getText(label, name);
+  }
+
+  getBackgroundColor = memoize((text, avatarColors, hashFunction, defaultColor) => {
+    return AvatarHelper.getBackgroundColor(text, avatarColors, hashFunction, defaultColor);
+  });
+
+  get backgroundColor() {
+    const {backgroundColor, useAutoColorsConfig, autoColorsConfig, name} = this.props;
+    if (backgroundColor) {
+      return backgroundColor;
+    }
+
+    const {
+      avatarColors = AvatarHelper.getAvatarColors(),
+      hashFunction = AvatarHelper.hashStringToNumber,
+      defaultColor = Colors.dark80
```

Comment:
dark preset should be deprecated eventually...
```suggestion
      defaultColor = Colors.grey80
```